wp-content/mu-plugins/locales.php

```diff
@@ -182,7 +182,6 @@ function locale_switcher() {
 	<div class="wp15-locale-switcher-container">
 		<form id="wp15-locale-switcher-form" action="" method="GET">
 			<label for="wp15-locale-switcher">
-				<span aria-hidden="true" class="dashicons dashicons-translation"></span>
```

Comment:
It could be beneficial to keep this. If someone only speaks their native language, then they might not understand the `English` prompt.

What do you think @marko-srb?

Reply:
I think it will at least need a new icon. That one is pretty dated. 

Since it appears like we'll have another iteration on design, can we consider this for that? I would like to merge this in the meantime.

Reply:
Yeah, that makes sense 👍🏻